### ISSUE TITLE
Add noob-friendly documentation on development setup

### ISSUE THREAD

**Issue by marklagendijk** [loadimpact/k6#428]
Most users of k6 are not familiar with programming Go. However, they _are_ developers, and as such potential code contributors.

I always say that if you can program, you can program in most languages. Usually the biggest hurdle to getting started is the programming environment. 

I think we should make it easier for Go noobs to get to the point they can send in a pull request.
Things we should document:

1. Installing the right Go version (1.9).  Either write or refer to [something like this](https://gist.github.com/d2s/6503f815431d1587c28bc37bfd715dbf).
2. Compiling the project: `go build`.
3. Running the tests: ?
4. Explain how you would use 'go get -u github.com/loadimpact/k6'. How does it relate to git cloning the repository?

**Comment by ivoreis** [comment]
@marklagendijk All good points. I would probably add dependency management too.

**Comment by liclac** [comment]
I'd love if someone could write something like this, preferably in the README. We used to have something like it, but scrapped it when we dropped the complications introduced by the web UI, under the assumption that there was nothing left to explain - a poor assumption indeed.

k6 development doesn't need any specific version of Go, we just try to track what the latest version is, and use godep for dependency management, but we could definitely use an explanation on how to install go and use `go get`, etc.
